tuxemon/resources/maps/downstairs_test.tmx

```diff
@@ -78,7 +78,7 @@
   </object>
   <object id="27" name="Go Outside" type="event" x="80" y="112" width="16" height="16">
    <properties>
-    <property name="act1" value="transition_teleport paper_town.tmx,31,7,0.3"/>
+    <property name="act1" value="transition_teleport map1.tmx,44,53,0.3"/>
```

Comment:
This action in the downstairs of the starting house appears to teleport the player to the wrong house. It should teleport the player to 43,46 instead.